ray/datadog_checks/ray/metrics.py

```diff
@@ -0,0 +1,111 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+METRIC_MAP = {
+    'process_cpu_seconds': 'process.cpu_seconds',
+    'process_max_fds': 'process.max_fds',
+    'process_open_fds': 'process.open_fds',
+    'process_resident_memory_bytes': 'process.resident_memory',
+    'process_start_time_seconds': 'process.start_time',
+    'process_virtual_memory_bytes': 'process.virtual_memory',
+    'python_gc_collections': 'python.gc.collections',
+    'python_gc_objects_collected': 'python.gc.objects_collected',
+    'python_gc_objects_uncollectable': 'python.gc.objects_uncollectable',
+    'ray_actors': 'actors',
+    'ray_cluster_active_nodes': 'cluster.active_nodes',
+    'ray_component_cpu_percentage': 'component.cpu_percentage',
+    'ray_component_mem_shared_bytes': 'component.mem_shared',
+    'ray_component_rss_mb': 'component.rss',
+    'ray_component_uss_mb': 'component.uss',
+    'ray_gcs_actors_count': 'gcs.actors',
+    'ray_gcs_placement_group_count': 'gcs.placement_group',
+    'ray_gcs_storage_operation_count': 'gcs.storage_operation',
+    'ray_gcs_storage_operation_latency_ms': 'gcs.storage_operation.latency',
+    'ray_gcs_task_manager_task_events_dropped': 'gcs.task_manager.task_events.dropped',
+    'ray_gcs_task_manager_task_events_reported': 'gcs.task_manager.task_events.reported',
+    'ray_gcs_task_manager_task_events_stored': 'gcs.task_manager.task_events.stored',
+    'ray_gcs_task_manager_task_events_stored_bytes': 'gcs.task_manager.task_events.stored_bytes',
+    'ray_grpc_server_req_finished': 'grpc_server.req.finished',
+    'ray_grpc_server_req_handling': 'grpc_server.req.handling',
+    'ray_grpc_server_req_new': 'grpc_server.req.new',
+    'ray_grpc_server_req_process_time_ms': 'grpc_server.req.process_time',
+    'ray_health_check_rpc_latency_ms': 'health_check.rpc_latency',
+    'ray_internal_num_infeasible_scheduling_classes': 'internal_num.infeasible_scheduling_classes',
+    'ray_internal_num_processes_skipped_job_mismatch': 'internal_num.processes.skipped.job_mismatch',
+    # There is a typo in their metric name, adding another entry to be ready as soon as they fix it
+    'ray_internal_num_processes_skipped_runtime_enviornment_mismatch': 'internal_num.processes.skipped.runtime_environment_mismatch',  # noqa: E501
```

Comment:
Question: is there a doc for these metrics? I know some are [system metrics](https://docs.ray.io/en/latest/ray-observability/reference/system-metrics.html#system-metrics) but not all. I was trying to find where the misspelled `ray_internal_num_processes_skipped_runtime_enviornment_mismatch` is documented so we can check that doc in the future to know it has been fixed.

Reply:
Most of the metrics are not documented. I had to take a look at the code to get them 😭

Reply:
Ah sad to hear 😢